modules/custom/az_event/az_event_trellis/az_event_trellis.info.yml

```diff
@@ -9,9 +9,11 @@ php: 8.0
 dependencies:
   - az_core
   - az_event
+  - drupal:views
   - migrate:migrate
   - migrate_plus:migrate_plus
   - migrate_tools:migrate_tools
   - migrate_queue_importer:migrate_queue_importer
```

Comment:
Maybe we don't need the migrate_queue_importer dependency anymore since this module has it's own `hook_cron()` implementation now (?).  If not, we could also probably change the migration to a migration plugin instead of config entity.
```suggestion
```

Reply:
I hadn't actually noticed until I was testing this out that there were migrate queue importer settings for this module - I think it might be better to use that, rather than hook cron implementation I'd previously committed. I'd like to discuss which would be preferable during PR review.

Reply:
We discussed this in PR review and leaned towards using `migrate_queue_importer` to handle the `cron` job.